plugin/pkg/scheduler/algorithm/well_known_labels.go

```diff
@@ -28,6 +28,26 @@ const (
 	// and removed when node becomes reachable (NodeReady status ConditionTrue).
 	TaintNodeUnreachable = "node.alpha.kubernetes.io/unreachable"
 
+	// When feature-gate for TaintBasedEvictions=true flag is enabled,
+	// TaintNodeOutOfDisk would be automatically added by node controller
+	// when node becomes out of disk, and removed when node has enough disk.
+	TaintNodeOutOfDisk = "node.alpha.kubernetes.io/outOfDisk"
+
+	// When feature-gate for TaintBasedEvictions=true flag is enabled,
+	// TaintNodeMemoryPressure would be automatically added by node controller
+	// when node has memory pressure, and removed when node has enough memory.
+	TaintNodeMemoryPressure = "node.alpha.kubernetes.io/memoryPressure"
+
+	// When feature-gate for TaintBasedEvictions=true flag is enabled,
+	// TaintNodeDiskPressure would be automatically added by node controller
+	// when node has disk pressure, and removed when node has enough disk.
+	TaintNodeDiskPressure = "node.alpha.kubernetes.io/diskPressure"
+
+	// When feature-gate for TaintBasedEvictions=true flag is enabled,
+	// TaintNodeNetworkUnavailable would be automatically added by node controller
+	// when node's network is unavailable, and removed when network becomes ready.
+	TaintNodeNetworkUnavailable = "node.alpha.kubernetes.io/networkUnavailable"
+
```

Comment:
@gmarek If it's deprecated we should state it somewhere - docs / [design proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/taint-node-by-condition.md) -. If not we should add it.